### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
## Changes I have worked on

I have changed some words from read me file and restructured the sentences for better understanding.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ This repo is part of the [Microsoft Bot Framework](https://github.com/microsoft/
 
 Starting from Web Chat 4.6.0, Web Chat requires React 16.8.6 or up.
 
-Although we recommend that you upgrade your host app at your earliest convenience, we understand that host app may need some time before its React dependencies are updated, especially in regards to huge applications.
+Although we recommend you to upgrade your host app at your earliest convenience, we understand that host app may need some time before its react dependencies are updated, especially in regards to huge applications.
```
Comment (callebdev) [inline README.md:25]: I don't think this is right, `React` is a noun, so it must start by a CAPITAL letter.

Comment (tdurnford) [reply README.md:25]: I agree with @callebdev.  I think both of your corrections to this sentence are unnecessary.

File: README.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ See the working sample of the [full Web Chat bundle](https://github.com/microsof
 
 For full customizability, you can use React to recompose components of Web Chat.
 
-To install the production build from NPM, run `npm install botframework-webchat`.
+To install the production build from NPM, run `npm install botframework-webchat` command.
```
Comment (tdurnford) [inline README.md:128]:
```suggestion
To install the production build from NPM, run the `npm install botframework-webchat` command.
```

**Comment by tdurnford** [comment]
@kaiwalyakoparkar Thank you for contributing to the Web Chat repository. We appreciate your feedback to the readme; however, we are currently happy with it's current state. If you notice any other issues in our repo, please feel free to open an issue or another PR, and we will be sure to take a look at it.

**Pull request closed without merging**